pkgs/misc/tmux-plugins/tmux-fingers/default.nix

```diff
@@ -0,0 +1,50 @@
+{
+  mkTmuxPlugin,
+  substituteAll,
+  fetchFromGitHub,
+  crystal,
+  shards,
+}:
+let
+  fingers = crystal.buildCrystalPackage rec {
+    format = "shards";
+    version = "2.2.2";
+    pname = "fingers";
+    src = fetchFromGitHub {
+      owner = "Morantron";
+      repo = "tmux-fingers";
+      rev = "${version}";
+      sha256 = "sha256-m9QON7diHVEDnnv/alXCJOG+BnfrAKygScrubZZ605I=";
+    };
+
+    shardsFile = ./shards.nix;
+    crystalBinaries.tmux-fingers.src = "src/fingers.cr";
+
+    postInstall = ''
+      shopt -s dotglob extglob
+      rm -rv !("tmux-fingers.tmux"|"bin")
+      shopt -u dotglob extglob
+    '';
+
+    doCheck = false;
+    doInstallCheck = false;
+
+    nativeBuildInputs = [
+      crystal
+      shards
+    ];
```

Comment:
Never used this builder, but usually these would be part of the default inputs. Is this not the case for `buildCrystalPackage`?

```suggestion
```